docs/docs/recipes.md

```diff
@@ -853,6 +853,129 @@ export const pageQuery = graphql`
+
+2. Create a new space. Make sure to save the space ID given to you at the end of the command.
+
+```shell
+contentful space create --name 'Gatsby example'
+```
+
+3. Seed the new space with example blog content using the new space ID returned from the previous command, in place of `<space ID>`.
+
+```shell
+contentful space seed -s '<space ID>' -t blog
+```
+
+For example, with a space ID in place: `contentful space seed -s '22fzx88spbp7' -t blog`
+
+4. Create a new access token for your space.
+
+```shell
+contentful space accesstoken create -s '<space ID>' --name 'Example token'
+```
+
+5. Install the `gatsby-source-contentful` plugin in your Gatsby site:
+
+```shell
+npm install --save gatsby-source-contentful
+```
+
+6. Edit the file `gatsby-config.js` to enable the `gatsby-source-contentful` plugin. You should strongly consider using [environment variables](/docs/environment-variables/) to store your space ID and token for security purposes.
+
+```javascript:title=gatsby-config.js
+plugins: [
+  {
+    resolve: `gatsby-source-contentful`,
+    options: {
+      spaceId: `<space ID>`, // or process.env.CONTENTFUL_SPACE_ID
+      accessToken: `<access token>`, // or process.env.CONTENTFUL_TOKEN
+    },
+  },
+]
+```
+
+7. Run `gatsby develop` and make sure the site compiled successfully.
+
+8. Query data with the [GraphiQL editor](/docs/introducing-graphiql/) at `https://localhost:8000/___graphql`. The Contentful plugin adds several new node types to your site, including every content type in your Contentful website. Your example space with a "Blog Post" content type produces a `allContentfulBlogPost` node type in GraphQL.
+
+![the graphql interface, with a sample query outlined below](./images/recipe-sourcing-contentful-graphql.png)
+
+To query for Blog Post titles from Contentful, use the following GraphQL query:
+
+```graphql
+{
+  allContentfulBlogPost {
+    edges {
+      node {
+        title
+      }
+    }
+  }
+}
+```
+
+Contentful nodes also include several metadata fields like `createdAt` or `node_locale`.
+
+9. To show a simple list of blog posts, create a new file in `/src/pages/blog.js`. This page will display all posts, sorted by updated date.
+
+```jsx:title=src/pages/blog.js
+import React from "react"
+import { graphql, Link } from "gatsby"
+
+const BlogPage = ({ data }) => (
+  <div>
+    <h1>Blog</h1>
+    <ul>
+      {data.allContentfulBlogPost.edges.map(({ node, index }) => (
+        <li key={index}>
+          <Link to={`/blog/${node.slug}`}>{node.title}</Link>
+        </li>
+      ))}
+    </ul>
+  </div>
+)
+
+export default BlogPage
+
+export const query = graphql`
+  {
+    allContentfulBlogPost(sort: { fields: [updatedAt] }) {
+      edges {
+        node {
+          title
+          slug
+        }
+      }
+    }
+  }
+`
+```
+
+To continue building out your Contentful site including post detail pages, check out the rest of the [Gatsby docs](/docs/sourcing-from-contentful/) and additional resources below.
```

Comment:
i missed a bonus -> the created page in browser :)

```suggestion
10. Open your browser and navigate to the new page: `http://localhost:8000/blog`. You see a list of links to the blog articles. If you click on a link you get the Gatsby 404 error page until you created the blog pages from a template (See tutorial/guide xxx)
(If the Gatsby development server is not running anymore restart it with `gatsby develop`.)

To continue building out your Contentful site including post detail pages, check out the rest of the [Gatsby docs](/docs/sourcing-from-contentful/) and additional resources below.
```

and i miss also the generated blog pages from contentful. maybe show this also here?

Reply:
Again -- recipes should be concise. Given this already has 10 steps, I would recommend limiting the scope and linking off for additional resources.

Reply:
ok - thats fine to wrap up the receipte to the result and then point to the next guide